spec/views/hyrax/admin/collection_types/_form_settings.html.erb_spec.rb

```diff
@@ -8,24 +21,47 @@
     end
   end
 
-  before do
-    assign(:form, collection_type_form)
-    allow(view).to receive(:f).and_return(form)
-    render
-  end
+  context "collection_type.collections? is false" do
+    before do
+      collection_type_form.collection_type = collection_type
+      assign(:form, collection_type_form)
+      allow(view).to receive(:f).and_return(form)
+      render
+    end
+
+    it "renders the intructions and warning" do
+      expect(rendered).to match(I18n.t("hyrax.admin.collection_types.form_settings.instructions"))
+      expect(rendered).to match(I18n.t("hyrax.admin.collection_types.form_settings.warning"))
+    end
+
+    INPUT_IDS.each do |id|
+      it "renders the #{id} checkbox to be disabled" do
+        match = rendered.match(/(<input.*id="#{id}".*>)/)
+        expect(match).not_to be_nil
+        next if match.nil?
```

Comment:
To avoid a Ruby error with the following line (`match[1]`) if `match` is indeed nil. May not be necessary?